### DISCUSSION TITLE
Is it possible to have a two-sided range?

### DISCUSSION THREAD

**Discussion by wsw70** [saadeghi/daisyui#2546] [Q&A] [answered]
The [range component](https://daisyui.com/components/range/) is great, but the documentation only shows a one-sided range (you change a single value, so it is effectively a range from "min value" to the one you set).

Is it possible to get two values with a double-sided slider (where you can set both "current left" and "current right" values, both between "min" and "max"?

**Comment by saadeghi** [comment, marked as answer]
See https://github.com/saadeghi/daisyui/issues/573#issuecomment-1064453159
